src/hooks/useFetch/useFetch.ts

```diff
@@ -9,10 +9,10 @@ export const useFetch = <TResponse>(url: string | undefined): FetchResult<TRespo
   useEffect(() => {
     const getData = async () => {
       try {
-        if (!url) {
+        if (!url && !(url === '')) {
```

Comment:
Dalej `url` może być równy `undefined`, dobrze by to było, żeby mógł przyjąć tylko wartość `string`